### ISSUE TITLE
Question regarding a particular hint

### ISSUE THREAD

**Issue by LU5GOON** [phrutis/LostWallets#1]
Hello there,

I have a question about a hint that i have pasted from your list below.

I just wanted to see if you know if it's just 11 small letters or is it a mixure of symbols, lower case letter & numbers?
is this info known? curious is all. 

![image](https://user-images.githubusercontent.com/100285625/193211175-68e76b09-a50e-430e-b1cb-895e85c845af.png)

Thank you for your time :) 

LU5GOON

**Comment by phrutis** [comment]
I don't believe in hints, they can be invented by the person who sells the files.
That is to say, marketing.

The hint matters if it comes from the owner, everything else is a fairy tale.

**Comment by LU5GOON** [comment]
Thank you for your reply.

So the hint you put up is basically speculation? It is not 100% correct?

**Comment by PhuocBull** [comment]
> Thank you for your reply.
> 
> So the hint you put up is basically speculation? It is not 100% correct?

Hints were gathered around the internet, on sites that are selling these wallets or from those who selling them.
Believe me, none of these hints are true, the dude that's trying to sell you the wallet he got from God knows where and made up some clues about the password with 6 or 7 characters, year 19xx at the end, starts with name "Dima" or "Neli",... yada yada, he's trying to trick you to think "oh this wallet is crackable, only 10 characters and have the year at the end, I just have to brute-force 6 other letters".
No, buddy. From 1 to 7 or 8 characters hints are definitely bullshit. I believe there are groups of people with enough GPU power out there that has done all the 1 to 8 brute-force on all these wallets already.

**Comment by LU5GOON** [comment]
> > Thank you for your reply.
> 
> > 
> 
> > So the hint you put up is basically speculation? It is not 100% correct?
> 
> 
> 
> Hints were gathered around the internet, on sites that are selling these wallets or from those who selling them.
> 
> Believe me, none of these hints are true, the dude that's trying to sell you the wallet he got from God knows where and made up some clues about the password with 6 or 7 characters, year 19xx at the end, starts with name "Dima" or "Neli",... yada yada, he's trying to trick you to think "oh this wallet is crackable, only 10 characters and have the year at the end, I just have to brute-force 6 other letters".
> 
> No, buddy. From 1 to 7 or 8 characters hints are definitely bullshit. I believe there are groups of people with enough GPU power out there that has done all the 1 to 8 brute-force on all these wallets already.

I very much value and respect your reply here. I couldn't agree more. I think the hints should be removed from this repo if no faith is present in it. It's quite misleading!

**Comment by PhuocBull** [comment]
> I think the hints should be removed from this repo if no faith is present in it. It's quite misleading!

Phrutis did say : "These are hints, if you believe them". Even himself doesn't believe in them.
For me, I'm glad that Phrutis put hints he found on here, it shows how far off the found correct password and will be found password are compared to the hints.

**Comment by phrutis** [comment]
Initially, I did not want to add them.
I have been asked many times to add them.
Added recently, now delete :)
Tomorrow they will start writing again, add tips.

**Comment by LU5GOON** [comment]
> Initially, I did not want to add them.
> 
> I have been asked many times to add them.
> 
> Added recently, now delete :)
> 
> Tomorrow they will start writing again, add tips.

At the end of the day it's your repo so do as you wish. I don't mind if it's there or not.
